backend/src/models/user.model.ts

```diff
@@ -2,8 +2,8 @@ import mongoose, {Schema} from 'mongoose';
 import bcrypt from 'bcrypt';
 
 const UserSchema: Schema = new Schema({
-  email: {type: String, required: true, unique: true},
-  password: {type: String, required: true},
+  email: { type: String, required: true, unique: true },
+  password: { type: String, required: true },
   images: [{ type: Schema.Types.ObjectId, ref: 'Image' }]
```

Comment:
you've already connected each image to appropriate users in images schema. i think that you can avoid referencing to image schema inside user schema.